app/controllers/admin/orders_controller.rb

```diff
@@ -0,0 +1,58 @@
+module Admin
+  class OrdersController < AdminController
+    @@users = User.all
```

Comment:
this would be set when the server starts up and would never change. Is this the intension?